ksm.go

```diff
@@ -0,0 +1,481 @@
+//
+// Copyright (c) 2017 Intel Corporation
```

Comment:
A thought, as this is effectively a new project - maybe we should start to use the SPDX copyright commenting method rather than the full blown long winded one?
https://spdx.org/licenses/
https://patchwork.kernel.org/patch/10016189/

It is becoming more common and accepted.

Reply:
Done